charts/posthog/values.yaml

```diff
@@ -344,6 +344,8 @@ ingress:
 postgresql:
   # -- Install postgres server on kubernetes (see below)
   enabled: true
+  # -- sslmode for connection - defaults to prefer
```

Comment:
Worth adding to the comment, what are the valid values?

Reply:
pretty easy to find on Google and that could change in theory if anything link to https://www.pgbouncer.org/config.html ?